consul/src/main/java/com/linecorp/armeria/internal/consul/ConsulClientBuilder.java

```diff
@@ -116,8 +118,8 @@ public ConsulClientBuilder consulApiVersion(String consulApiVersion) {
         checkState(consulUri == null, "consulApiVersion can't comes with consulUri");
         requireNonNull(consulApiVersion, "consulApiVersion");
         checkArgument(!consulApiVersion.isEmpty(), "consulApiVersion can't be empty");
-        checkArgument(consulApiVersion.charAt(0) != '/',
-                      "consulApiVersion can't starts with '/'");
+        checkArgument(CONSUL_VERSION_PATTERN.matcher(consulApiVersion).matches(),
+                      "consulApiVersion has unexpected format");
```

Comment:
`"consulApiVersion: %s (expected: a version string that starts with 'v', e.g. 'v1')", consulVersion`

Reply:
Ok.